### PR TITLE
Proofread types doc

### PR DESCRIPTION
**Pull request by greg0ire**
This should help making #2839 look less messy, and is far more mergeable.

### CODE REVIEW

**Comment by greg0ire** [comment]
Triggering a new build

File: docs/en/reference/types.rst [hunk 1]
```diff
@@ -927,7 +927,10 @@ Now we implement our ``Doctrine\DBAL\Types\Type`` instance:
         }
     }
 
-The job of Doctrine-DBAL is to transform your type into SQL declaration. You can modify the SQL declaration Doctrine will produce. At first, you must to enable this feature by overriding the canRequireSQLConversion method:
+The job of Doctrine-DBAL is to transform your type into SQL declaration.
+You can modify the SQL declaration Doctrine will produce. First, you
```
Comment (Ocramius) [inline docs/en/reference/types.rst]: that Doctrine

File: docs/en/reference/types.rst [hunk 2]
```diff
@@ -927,7 +927,10 @@ Now we implement our ``Doctrine\DBAL\Types\Type`` instance:
         }
     }
 
-The job of Doctrine-DBAL is to transform your type into SQL declaration. You can modify the SQL declaration Doctrine will produce. At first, you must to enable this feature by overriding the canRequireSQLConversion method:
+The job of Doctrine-DBAL is to transform your type into SQL declaration.
```
Comment (Ocramius) [inline docs/en/reference/types.rst]: And to convert values?

Comment (Ocramius) [reply docs/en/reference/types.rst]: Or is that covered in the rest of this article?

File: docs/en/reference/types.rst [hunk 3]
```diff
@@ -937,7 +940,8 @@ The job of Doctrine-DBAL is to transform your type into SQL declaration. You can
         return true;
     }
 
-Then you override the methods convertToPhpValueSQL and convertToDatabaseValueSQL :
+Then you override the methods ``convertToPhpValueSQL`` and
```
Comment (Ocramius) [inline docs/en/reference/types.rst]: `methods` to be put at the end of this sentence

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: docs/en/reference/types.rst (hunk 2 above)
Comment (greg0ire) [reply docs/en/reference/types.rst]: It is covered just below, on line 943

**Comment by greg0ire** [comment]
@Ocramius fixed, please review again

**Comment by greg0ire** [comment]
continuousphp seems to be stuck... should I close and reopen?

**Comment by lcobucci** [comment]
@greg0ire just amending should restart continuousphp

**Comment by greg0ire** [comment]
Clicking 2 buttons sounds easier :P

**Comment by greg0ire** [comment]
It's unstuck :)

**Comment by greg0ire** [comment]
ping  @Ocramius

**Review by Majkl578 — APPROVED** [review]
👍, maybe a single squashed commit instead of 6 would be cleaner though... 😇

**Comment by Ocramius** [comment]
Plz don't squash stuff unless needed 😅

**Comment by greg0ire** [comment]
@Majkl578 if the PR is easier to read like that then no squash is needed :P

**Comment by SenseException** [comment]
I've got nothing to add for this PR. :+1:

**Comment by greg0ire** [comment]
Rebased.

**Review by Ocramius — APPROVED** [review]
LGTM, thanks @greg0ire!

**Pull request merged**